README.md

```diff
@@ -163,87 +178,22 @@ To further explain the various environment variables and what they do:
+| `WWW_CONTENT_PATH` | `www` | `/path/to/ocw-content-rc/ocw-www` | A path to a Hugo site that will be rendered when running `yarn start www` |
 | `COURSE_CONTENT_PATH` | `course` | `/path/to/ocw-content-rc/` | A path to a base folder containing `ocw-course` type Hugo sites |
-| `OCW_TEST_COURSE` | `course` | `18.06-spring-2010` | The name of a folder in `COURSE_CONTENT_PATH` containing a Hugo site that will be rendered when running `npm run start:course` |
+| `OCW_TEST_COURSE` | `course` | `18.06-spring-2010` | The name of a folder in `COURSE_CONTENT_PATH` containing a Hugo site that will be rendered when running `yarn start course` |
 | `OCW_COURSE_STARTER_SLUG` | `www` | `ocw-course` | When generating "New Courses" cards on the home page, the `ocw-studio` API is queried using `OCW_STUDIO_BASE_URL`.  This value determines the `type` used in the query string against the API |
 | `FIELDS_HUGO_CONFIG_PATH` | `fields` | `/path/to/ocw-hugo-projects/mit-fields/config.yaml` | A path to the `mit-fields` Hugo configuration file |
-| `FIELDS_CONTENT_PATH` | `fields` | `/path/to/ocw-content-rc/philosophy` | A path to a Hugo site that will be rendered when running `npm run start:fields` |
+| `FIELDS_CONTENT_PATH` | `fields` | `/path/to/ocw-content-rc/philosophy` | A path to a Hugo site that will be rendered when running `yarn start fields` |
 | `WEBPACK_ANALYZE` | N/A | `true` | Used in webpack build. If set to `true`, a dependency analysis of the bundle will be included in the build output. |
 | `WEBPACK_HOST` | N/A | `localhost` | Host used by Hugo when querying the Webpack Dev Server. Can be set to your local IP to enable testing OCW on other devices (e.g., phones) within your network. |
 | `WEBPACK_PORT` | N/A | `3001` | Port used by Webpack Dev Server |
 
-### Running ocw-www
-
-To run `ocw-www` locally for working on the `www` theme:
-
-`npm run start:www`
-
-To customize your `www` site:
-
- - Clone the `ocw-www` content repo: https://github.mit.edu/ocw-content-rc/ocw-www
- - Clone `ocw-hugo-projects` to obtain the relevant configuration file: https://github.com/mitodl/ocw-hugo-projects
- - Set the following environment variables in your `.env` file, replacing `/path/to` with your path to the repos indicated:
-   - `WWW_HUGO_CONFIG_PATH=/path/to/ocw-hugo-projects/ocw-www/config.yaml`
-   - `WWW_CONTENT_PATH=/path/to/ocw-content-rc/ocw-www/`
- - Optionally set these environment variables as well, depending on the functionality you need to work on:
-   - `SEARCH_API_URL=https://discussions-rc.odl.mit.edu/api/v0/search/` (for testing search functionality)
-   - `OCW_STUDIO_BASE_URL=http://ocw-studio-rc.odl.mit.edu/` (for testing `ocw-studio` API functionality, such as the "new courses" section rendered by `www/layouts/partials/home_course_cards.html`)
-   - `RESOURCE_BASE_URL=https://live-qa.ocw.mit.edu/` (if you need to test the loading of resources from S3 or some other CDN)
```

Comment:
The Readme probably should have been revisited as part of #964, but I'm glad it waited till this PR. (Because now the CLI commands have --help). 

I thought the 3 "Running <xyz>" portions of the readme were a little redundant, so I combined them into "### Running Sites (courses and ocw-www)". Additionally, the instructions to set env variables are generally unnecessary now.